index.md

```diff
@@ -1,63 +1,200 @@
+high-quality scrolling images, interactive cards, buttons and dropdown lists, 
+and a shopping cart page. Run Shrine on an iPhone or Android phone, then check
+out the single codebase for this and more examples [here](https://github.com/flutter/flutter/tree/master/examples) 
+in our github repository.
+
+No mobile development experience is required to get started. Apps are written 
+in [Dart](https://dartlang.org), which will look familiar if you've used a 
+language like Java or JavaScript. Experience with object-oriented languages 
+is definitely helpful, but even non-programmers have made Flutter apps!
+
+## Why use Flutter?
+
+What are some advantages of Flutter? It helps you:
 
-See these topics to understand how the Flutter framework works.
-
- - [Widgets overview](widgets)
- - [Gestures in Flutter](gestures)
- - [Animations in Flutter](animations)
- - [Using custom fonts](custom-fonts)
- - [Layered design (video)](https://www.youtube.com/watch?v=dkyY9WCGMi0)
- - [Rendering pipeline (video)](https://www.youtube.com/watch?v=UUfXWzp0-DU)
- - [Dealing with box constraints in Flutter](layout)
-
-## How-Tos
-
-Learn how to accomplish specific development tasks with Flutter.
-
- - [Building Layouts in Flutter](/tutorials/layout/)
- - [Adding Assets and Images](assets-and-images)
- - [Testing Flutter Apps](testing)
- - [Debugging Flutter Apps](debugging)
- - [Developing apps in the IntelliJ IDE](intellij-ide)
- - [Formatting Flutter code](formatting)
- - [Upgrading Flutter](upgrading)
- - [Accessing Platform and Third-Party Services](platform-services)
- - [Reading and Writing Files](reading-writing-files)
- - [Handling Text Input](text-input)
-
-## Helpful Resources
-
-More resources for Flutter developers.
-
- - [Source on GitHub](https://github.com/flutter/flutter)
- - [Architecture diagram](https://docs.google.com/presentation/d/1cw7A4HbvM_Abv320rVgPVGiUP2msVs7tfGbkgdrTy0I/edit?usp=sharing)
- - [Issue tracker](https://github.com/flutter/flutter/issues)
- - [Writing useful bug reports](bug-reports)
- - [FAQ](faq)
- - [Design principles](design-principles)
-
-## Support
+*   Be highly productive
+    *   Develop for iOS and Android from a single codebase
+    *   Do more with less code, even on a single OS, with a modern, expressive 
+language and a declarative approach
+    *   Prototype and iterate easily
+        *   Experiment by changing code and reloading as your app runs (with hot reload)
+        *   Fix crashes and continue debugging from where the app left off
+*   Create beautiful, highly-customized user experiences
+    *   Benefit from a rich set of Material design widgets built using
+        Flutter's own framework
+    *   Realize custom, beautiful, brand-driven designs, without the limitations 
+of OEM widget sets
+
+## Core principles
+
+Flutter includes a functional-reactive framework, a 2D rendering engine, ready-
+made widgets, and development tools. The components work together to help you 
```

Comment:
Will ready-made show up on the site without a space after the - even though there's a line break there?